### PR TITLE
Issue159-Unit Testing: GridHeader

### PR DESCRIPTION
**Pull request by ArCiGoUnoSquare**
Continuing with the past branch [Issue157-Unit testing: GridHeader](https://github.com/unosquare/tubular-react/pull/158): 

* Test cases added to check the functionality of the component
* The `<GridHeader />` component was refactored
* Was checked the functionality of the `<DataGrid />` component to render `boolean` data and was added a function to render the `<body />`
* Validations added on `exportCSV()` and `printTable` to render `boolean` data

### CODE REVIEW

File: src/DataGrid/GridHeader.tsx [hunk 1]
```diff
@@ -233,7 +201,7 @@ class GridHeader extends React.Component <IProps & WithStyles<keyof typeof style
     document.removeEventListener('keyup', (event) => this.handleKeyUp(event));
   }
 
-  public sortHandler = (event: React.MouseEvent<HTMLElement>, property: any) => {
+  public sortHandler = (property: any) => {
```
Comment (kadosh) [inline src/DataGrid/GridHeader.tsx]: Please define a type for "property".

File: src/DataGrid/GridHeader.tsx [hunk 2]
```diff
@@ -258,7 +226,7 @@ class GridHeader extends React.Component <IProps & WithStyles<keyof typeof style
         }
 
         const currentlySortedColumns = array.columns.filter((col: any) => col.SortOrder > 0)
```
Comment (kadosh) [inline src/DataGrid/GridHeader.tsx]: What about types for "col"? for "a" and "b".

File: src/DataGrid/GridHeader.tsx [hunk 3]
```diff
@@ -258,7 +226,7 @@ class GridHeader extends React.Component <IProps & WithStyles<keyof typeof style
...
-        .sort((a: any, b: any) => a.SortOrder === b.SortOrder ? 0 : a.SortOrder > b.SortOrder );
+          .sort((a: any, b: any) => a.SortOrder === b.SortOrder ? 0 : a.SortOrder > b.SortOrder );
 
         currentlySortedColumns.forEach( ($column: any, i: number) => {
```
Comment (kadosh) [inline src/DataGrid/GridHeader.tsx]: Types please...

File: src/DataGrid/DataGrid.tsx [hunk 4]
```diff
@@ -245,6 +252,29 @@ class DataGrid extends React.Component<IProps & WithStyles<keyof typeof styleCla
       });
   }
 
+  public columnRendering = (column: any, row: any) => {
```
[the remark this answers is not in the document]
Comment (ArCiGoUnoSquare) [reply src/DataGrid/DataGrid.tsx]: Done

File: src/DataGrid/GridHeader.tsx (hunk 1 above)
Comment (ArCiGoUnoSquare) [reply src/DataGrid/GridHeader.tsx]: Done

**Pull request merged**